### PR TITLE
[TECH-513] Update usage readme

### PR DESCRIPTION
**Pull request by koeves**
branch: feature/TECH-513_quick_setup_tutorial

----
### 📕 [TECH-513](https://vandebron.atlassian.net/browse/TECH-513) Create/centralise a quick setup tutorial danielkoves@vandebron.nl 
Currently information on how to quickly “run” mpyl is scattered around in the pip library description, github readme and `mpyl health` commands.

Centralise them and create a short tutorial and how to start from scratch with mpyl and deploy a PR.

----

Mention MPyL in the monorepo README and link this tutorial there

🏗️ Build [3](https://jenkins.k8s-serv-backend.vdbinfra.nl/job/MPyL%20Pipeline%20-%20Test/job/PR-234/3/display/redirect) ✅ Successful, started by _Sam Theisens_  
🚀 *[cloudfront-service](https://cloudfront-service-234.test.nl/)*, *example-dagster-user-code*, *job*, *kong-sync*, *[nodeservice](https://nodeservice-234.test.nl/)*, *[sbtservice](https://sbtservice-234.test.nl/)*, *sparkJob*  

[TECH-513]: https://vandebron.atlassian.net/browse/TECH-513?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README-usage.md [hunk 1]
```diff
@@ -2,22 +2,34 @@
...
 ```shell
 mpyl health
 ```
 Will validate the configuration and check if all required tools are installed.
 
-##### Command structure
+#### 3. Create a PR with your changes
+
+```shell
+gh pr create --draft
```
Comment (Jorg88) [inline README-usage.md:23]: Do we mention the gh cli requirement anywhere?

**Review by Jorg88 — APPROVED** [review]

File: README-usage.md (hunk 1 above)
Comment (koeves) [reply README-usage.md:23]: It's mentioned in mpyl health

File: README-usage.md [hunk 2]
```diff
@@ -2,22 +2,34 @@
...
+```
+
+#### 4. Trigger an MPyL build via the CLI
+
+```shell
+mpyl build jenkins
+```
```
Comment (SamTheisens) [inline README-usage.md:30]: Maybe we could give options here. If you use GHA, the build will probably be triggered automatically. And you may also want to run a local build.

File: README-usage.md (hunk 1 above)
Comment (SamTheisens) [reply README-usage.md:23]: Perhaps we could mention somewhere that this a _recommended_ workflow.

**Review by SamTheisens — APPROVED** [review]

**Review by Jorg88 — APPROVED** [review]

**Pull request merged**